pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -436,6 +436,7 @@ func (qjm *XController) PreemptQueueJobs() {
 			message = fmt.Sprintf("Insufficient number of Running and Completed pods, minimum=%d, running=%d, completed=%d.", aw.Spec.SchedSpec.MinAvailable, aw.Status.Running, aw.Status.Succeeded)
 			cond := GenerateAppWrapperCondition(arbv1.AppWrapperCondPreemptCandidate, v1.ConditionTrue, "MinPodsNotRunning", message)
 			newjob.Status.Conditions = append(newjob.Status.Conditions, cond)
+			newjob.Spec.SchedSpec.RequeuingTimeMinutes = 2 * aw.Spec.SchedSpec.RequeuingTimeMinutes // Grow the requeuing waiting time exponentially
```

Comment:
use requeueCount to enable max requeue retries, suggest not exposing this field to the user for now

Reply:
Done. I decided to expose it to the users but it has a default value to disable it, so only people who know what they're doing can set it.